conda-nighres.yml

```diff
@@ -1,36 +1,10 @@
 name: nighres
 channels:
-  - mrtrix3
-  - simpleitk
-  - itk
   - conda-forge
-  - bioconda
-  - defaults
```

Comment:
AFAICT only conda-forge is needed.

I checked locally and on the CIO of a PR on my fork: this still builds.

Reply:
Do all the tests run?

Reply:
https://github.com/nighres/nighres/tree/master/examples

Reply:
Added tests in CI they are running on my fork:

See for example: https://github.com/Remi-Gau/nighres/runs/5758099723?check_suite_focus=true

As it will run all the example, this could take a while so it might be good to try to find a way to run each example in parallel.

Reply:
Yeah the examples are computationally expensive, I tested them for the "does this conda work" not sure a CI will allow these given how long they take.

Reply:
Apparently a workflow can run up to 72 hours. So it should be enough but not sure that we'd want to start a new workflow on every push. Things could get ugly.

https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration

Reply:
We could have a separate test/ dataset (basically the example/ data subsampled *a lot*), but it is important for the examples to run on high resolution data. It might still take a couple of hours to run through all, so maybe better to set up different tests focused on checking that dependencies and such are properly installed?

Reply:
Why not both? :-)

Good to have system level tests and infra or unit tests.

Reply:
Suggestion: 

Github CI accepts CRON jobs to run some worklows at regular intervals.

I think that it would make sense to run the conda set up and the system level smoke tests every 15 days or every month rather than on every push or pull request.

If that workflow eventually fails then one "only" has to look into the changes made in the past X days to figure out where the problem was introduced.

If the traffic and number of PRs increases in frequency then those long tests could be run more frequently.

@piloubazin @gdevenyi WDYT?

Reply:
Done